pyDeltaRCM/init_tools.py

```diff
@@ -113,17 +116,13 @@ def set_constants(self):
                               [-1, 0, 1],
                               [-sqrt05, 0, sqrt05]])
 
-        self.iwalk = np.array([[-1, 0, 1],
-                               [-1, 0, 1],
-                               [-1, 0, 1]])
+        self.iwalk = shared_tools.get_iwalk()
 
         self.jvec = np.array([[-sqrt05, -1, -sqrt05],
```

Comment:
If `iwalk` and `jwalk` are moved to `shared_tools` (which is fine by me) does it make sense to also move over all of the other walk/vector arrays like `jvec` and so forth?

Reply:
Yeah, I think I see a larger 'reorganize and tidy' PR on the horizon, where we once again move stuff into separate classes. I think this relates to the milestones for splitting out the hydrodynamics and morphodynamics code into separate modular classes, with a `utils` or similar module that they both access. I think moving a bunch of the stuff like the definitions of these arrays into something like `utils` makes a lot of sense.